### DISCUSSION TITLE
Avoir la possibilité de configurer la propriété clip-path polygon dans les acteurs

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#730] [Ideas]
Bonjour,

Serait'il possible d'ajouter dans les bordures ou les effets la propriété clip-path: polygon( ) ?

Merci d'avance
Jérémie DENAEGEL
GAEM

**Comment by Raphwit** [comment]
Hello, nous considérons que cette propriété est un usage un peu plus avancée pour le moment, c'est pourquoi nous ne l'avons pas ajoutée aux propriétés "de base".
Il est cependant toujours possible de l'utiliser en incluant du CSS dans un acteur HTML comme nous le faisons souvent.

On garde cependant la demande en tête, nous pourrions éventuellement l'ajouter à l'avenir, rien n'est exclu 😀
